fms/modules/feedforward.py

```diff
@@ -147,6 +164,95 @@ def load_weights(
             self.sharded_copy(self.w1.bias, w1_bias, 0, [self.world_size])
             self.sharded_copy(self.w2.bias, w2_bias, 1, [self.world_size], False)
 
+    def load_qparams(
```

Comment:
see previous comment

Reply:
Is this generalizable? Do we need 2 similar implementations for qkv and mlp separately?